app/views/charge.njk

```diff
@@ -27,7 +32,7 @@
     </div>
 
   {% if allowApplePay or allowGooglePay %}
-    <form id="payment-request-container" class="hidden">
+    <form class="web-payments-container">
```

Comment:
Why did this change from being an ID to a class?

Reply:
ID’s are good for JS to hook into, but bad (and slower) to use in CSS, so I switched

Reply:
I thought it was the other way round…

https://csswizardry.com/2011/09/writing-efficient-css-selectors/